### ISSUE TITLE
Read automerge configuration from Tide config.

### ISSUE THREAD

**Issue by dekiel** [kyma-project/test-infra#5210]
**Description**

To allow Tide merge PRs without review, a dedicated queries are defined in configuration. Automerge-notification plugin send notifications for PR which were automatically merged. That means plugin react on PRs that match the same criteria. Automerge-notification plugin should use a Tide config to get criteria for sending notification.

**Reasons**

Having hardcoded or separate configuraion of automerge-notification criteria will cause a configuration drift and require additional work every time Tide config will change.

**Acceptance Criteria**

Automerge-notification plugin send notification for PRs that match Tide criteria for skipping review.

**Comment by Ressetkk** [comment]
No, I would go the way as simple as possible. We will define a query is valid for merging only when a label `skip-review` is added. It will be valid only for @kyma-bot and dependabot. I would go with filtering the notification only by a given set of labels. If the PR with label `skip-review` is merged - you get a notification, because those would be unattended merges.

**Comment by dekiel** [comment]
This approach require use of Prow approval plugin which we doesn't use anymore.
